specs/core/0_beacon-chain.md

```diff
@@ -464,16 +461,22 @@ The following data structures are defined as [SimpleSerialize (SSZ)](https://git
 
 ```python
 {
-    # Header
     'slot': 'uint64',
-    'parent_root': 'bytes32',
+    'previous_block_root': 'bytes32',
     'state_root': 'bytes32',
-    'randao_reveal': 'bytes96',
-    'eth1_data': Eth1Data,
-
-    # Body
     'body': BeaconBlockBody,
-    # Signature
+    'signature': 'bytes96',
+}
+```
+
+#### `BeaconBlockHeader`
+
+```python
+{
+    'slot': 'uint64',
+    'previous_block_root': 'bytes32',
```

Comment:
why `previous_` instead of `parent_`?

Reply:
Consistency—we use `previous_` everywhere else (`previous_epoch`, `previous_version `, `previous_shuffling_start_shard`, etc.)